### PR TITLE
[HMS-2031] feat: configuration for Directory and Domain Service

### PR DESCRIPTION
**Pull request by avisiedo**
Add the configuration to make available "Directory & Domain Services" at stage beta environment. The micro-frontend will be accessible from the Settings section, and "Directory & Domain Services" will be the entry to display.

### CODE REVIEW

**Comment by avisiedo** [comment]
Happy to get a first round of review if everything is ok, or something is missing before merging. Thanks in advance.

**Comment by avisiedo** [comment]
sure! currently it is here: https://gitlab.cee.redhat.com/identity-management/idmocp/idm-domains-frontend

File: static/beta/stage/modules/fed-modules.json [hunk 1]
```diff
@@ -985,6 +985,21 @@
             }
         ]
     },
+    "idmsvc": {
```
Comment (Hyperkid123) [inline static/beta/stage/modules/fed-modules.json:988]: Your appname is `hmsidm` in your source files here: https://gitlab.cee.redhat.com/identity-management/idmocp/idm-domains-frontend/-/blob/main/package.json#L75

The key has to match the `appname`. The same goes for the route to the manifest file.

**Review by Hyperkid123 — CHANGES_REQUESTED** [review]

File: static/beta/stage/modules/fed-modules.json (hunk 1 above)
Comment (Hyperkid123) [reply static/beta/stage/modules/fed-modules.json:988]: I see, then we will have to wait before we can merge.

Comment (avisiedo) [reply static/beta/stage/modules/fed-modules.json:988]: :+1: thanks mate for reviewing this and your time! if any further change could be necessary, more than happy to make it.

**Comment by avisiedo** [comment]
@Hyperkid123 hi mate :smiley:  I think I am ready for a second round!  several IDs were aligned.

**Review by Hyperkid123 — COMMENTED** [review]
The configuration seems correct now. However, I don't see the actual assets to be deployed in the stage beta env. Is that expected?

**Comment by avisiedo** [comment]
@Hyperkid123 I am afraid not! I thought this configuration was necessary for ephemeral! by your comment I think I was wrong about it! right?

**Comment by Hyperkid123** [comment]
@avisiedo it partially is, but you also have to make sure that your frontend operator build pipeline is also setup. As it is used in EE as well.

File: static/beta/stage/navigation/settings-navigation.json [hunk 2]
```diff
@@ -45,6 +45,12 @@
                 }
             ]
         },
+        {
+            "id": "idmsvc",
+            "appId": "idmsvc",
+            "title": "Directory & Domain Services",
+            "href": "/settings/idmsvc"
```
Comment (Hyperkid123) [inline static/beta/stage/navigation/settings-navigation.json:52]: Can you add a short description to the link? We require descriptions for links that are exposed in the services dropdown and by extension in the search results.

Comment (avisiedo) [reply static/beta/stage/navigation/settings-navigation.json:53]: @Hyperkid123 sure! thank you mate! done

I have rebased and squashed the commits.

**Pull request merged**